rrdtool/rrd_mts_similarity.c

```diff
@@ -58,22 +59,22 @@ u_int verbose = 0, similarity_threshold = 100, skip_zero = 0;
 /* *************************************************** */
 
 static void help() {
-  printf("Usage: rrd_similarity [-v][-a <alpha>][-e <end>][-q][-s <start>]\n"
-	 "                      -f <filename> -d <basedir> [-t <threshold>]\n"
+  printf("Usage: rrd_mts_similarity [-v][-a <alpha>][-e <end>][-q][-s <start>]\n"
+	 "                      -f <filename_1> <filename_2> ... <filename_n> -d <basedir> [-t <threshold>]\n"
 	 "-a             | Set alpha. Valid range >0 .. <1. Default %.2f\n"
 	 "-e <end>       | RRD end time. Default %s\n"
 	 "-q             | Quick output (only anomalies are reported)\n"
 	 "-s <start>     | RRD start time. Default %s\n"
 
 	 "-d <basedir>   | Base directory where RRD filename is searched\n"
-	 "-f <rrd path>  | Path of the RRD filename to analyze\n"
+	 "-f <rrd_path1> <rrd_path2> ...  | Path of the RRDs filename to analyze\n"
 	 "-t <threshold> | Similarity threshold. Default %u (0 == alike)\n"
 	 "-v             | Verbose\n"
 	 "-z             | Skip zero RRDs during comparison\n"
 	 ,
 	 DEFAULT_ALPHA, DEFAULT_END, DEFAULT_START, similarity_threshold);
 
-  printf("\n\nExample: rrd_similarity -q -f bytes.rrd -d /var/lib/ntopng/-1/snmpstats\n");
+  printf("\n\nExample: rrd_mts_similarity -q -f bytes.rrd score.rrd -d /var/lib/ntopng/-1/snmpstats\n");
```

Comment:
updated example 
from rrd_similarity to rrd_mts_similarity
from -f bytes.rrd to -> -f bytes.rrd score.rrd

Reply:
Without " "  (quotes) it cannot work

Reply:
Reworked -f in commit 108daa5